changelog/_unreleased/2025-04-30-add-deprecated-prop-to-facilitate-the-migration-to-meteor-components.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Add deprecated prop to facilitate the migration to meteor components
+issue: 8859
+author: Iván Tajes Vidal
+author_email: tajespasarela@gmail.com
+author_github: @Iván Tajes Vidal
+---
+# Administration
+* Added `deprecated` prop to all components that are being replaced by Meteor Components.
+___
+# Upgrade Information
+## Use `deprecated` prop
+This prop is used to use deprecated components to keep the same API and not breaking previous code. This way the same code is compatible with Shopware 6.6 and Shopware 6.7. This prop will be removed in the next major version.
```

Comment:
Question: won't the wrapper components be removed completely? Then we should probably mention that to make sure people know that this is coming

Additionally let's make it explicit and say that the use of the deprecated prop "works" in 6.6 as well as it is just ignored, so I would mention that explicitly with a sentence and maybe adjust the docs in the code snippets to say: uses mt-button on 6.7 and sw-button-deprecated in 6.6 (without prop) and then with prop: uses sw-button-deprecated in both 6.6 and 6.7